plugins/uberdev/skills/testers-pipeline/aggregate.py

```
@@ -0,0 +1,88 @@
+
+
+def main() -> int:
+    p = argparse.ArgumentParser()
+    p.add_argument("--run-id", required=True)
+    p.add_argument("--wave", required=True, type=int)
+    p.add_argument("--scratch-dir", required=True)
+    p.add_argument("--invariants", required=True)
+    p.add_argument("--out", required=True)
+    args = p.parse_args()
+
+    invariants = {i["id"] for i in yaml.safe_load(open(args.invariants))["invariants"]}
```

Ruff (0.15.13)

[warning] Use a context manager for opening files

(SIM115)